internal/core/services/operation_manager/operation_manager.go

```diff
@@ -245,6 +245,11 @@ func (om *OperationManager) StartLeaseRenewGoRoutine(operationCtx context.Contex
 					break renewLeaseLoop
 				}
```

Comment:
```suggestion
	if op.Status == operation.StatusFinished ||  op.Status == operation.StatusError{
					zap.L().Info(fmt.Sprintf("finish operation lease renew go routine since operation got status %s"), op.Status)
					break renewLeaseLoop
				}
```

Reply:
@caiordjesus